lib/scp.h

```diff
@@ -27,17 +27,24 @@ enum
     SCP_FLAG_FOOTER     = (1<<5)
 };
 
+struct ScpTrackStart
+{
+    char track_id[3];      // 'TRK'
+    uint8_t strack;        // SCP track number
+};
+
+struct ScpTrackRevolution
+{
+    uint8_t index[4];  // time for one revolution
+    uint8_t length[4]; // number of bitcells
+    uint8_t offset[4]; // offset to bitcell data, relative to track header
+};
+
 struct ScpTrack
 {
     char track_id[3];      // 'TRK'
```

Comment:
Could probably be a ScpTrackHeader here.

Reply:
Although, taking another look at it, doesn't your change mean that ScpTrack is never used any more? If so, remove it, please?

Reply:
ScpTrack is still used by scpfluxsink. I swapped to ScpTrackHeader here and updated scpfluxsink.

I didn't earlier either to avoid (minor) changes to the sink, or to avoid alignment concerns (yes, it happens to work fine in this case because ScpTrackHeader is 4 bytes, but that stuff is subtle). Don't recall which.